.github/workflows/arduino-finsh-line-CI.yml

```diff
@@ -0,0 +1,26 @@
+name: Arduino Finish Line CI
+
+on: [push, pull_request]
+
+jobs:
+  build-sketch:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v2
+      - name: Install Arduino CLI
+        uses: arduino/setup-arduino-cli@v1.1.1
+      - name: Get Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Fetch sketch dependencies
+        run: |
```

Comment:
There's no reason to keep this as a separate workflow. You already have a workflow for Arduino sketches, why not reuse that instead?

Reply:
Something like this can may solve this.
  in   - name: Build sketch
        run: |
          arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1 arduino/finish_line/finish_line.ino && arduino/smartcar/smartcar.ino

or maybe           
 arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1 arduino/finish_line/finish_line.ino && arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1arduino/smartcar/smartcar.ino

Reply:
@Quinstedt 
The first one doesn't work, the second one is correct, however, no need for `&&`, just put the second command in a new line.

Reply:
Don't forget to remove this file